### ISSUE TITLE
A11y

### ISSUE THREAD

**Issue by Latha2564** [microsoft/vscode-azurefunctions#3395]
**GitHubTags:**

#A11ySev4; #TCS-E+d; #NFT-Fluentui-Web-Oct2022; #DesktopWeb; #WCAG2.4.7.
#Keyboard; #A11yMAS,#Usable

**OS Details:**  

Web: win11

**Environment Details:** 

Application Name: Fluent Ui

**Repro Steps:** 

1.Hit enter on the URL https://react-dev.fluentui.dev/
2."Fluent UI React v9" page will open.
3.Tab till "Components" button and hit enter.
4.Tab till "Accordion" link on the left side bar and hit enter.
5.Turn on high contrast mode
6.verify in high contrast mode keyboard focus is not visible.

**Actual Result:** 

In high contrast mode keyboard focus is not visible but tooltip is visible
Similar issues are observed in the whole application.

**Expected Result:**  

when we apply high contrast mode, keyboard focus should also be visible clearly 

**User Impact:** 

keyboard users will have difficulties In high contrast mode keyboard focus is not visible, but tooltip is visible

![3395A11y_Fluent UI_Components_accordion_keyboard](https://user-images.githubusercontent.com/89508906/198254524-089ef5e6-4694-4ba3-99b0-9ee69fb4ed7e.gif)

**Comment by Latha2564** [comment]
Bug is closing as the buug is logge din wrong ADO
